libr/bin/mangling/swift-sd.c

```diff
@@ -217,8 +234,6 @@ R_API char *r_bin_demangle_swift(const char *s, bool syscmd, bool trylib) {
 		}
 	}
 
-	out[0] = 0;
```

Comment:
uh?

Reply:
out changed from an array to an `RStrBuf`. Doesn't `r_strbuf_drain(r_strbuf_new (NULL))`  already result in `"\0"`?